Sources/Afterpay/Model/Brand.swift

```diff
@@ -0,0 +1,63 @@
+//
+//  Brand.swift
+//  Afterpay
+//
+//  Created by Scott Antonac on 25/3/2022.
+//  Copyright © 2022 Afterpay. All rights reserved.
+//
+
+import Foundation
+
+private let clearpayLocales: Set<Locale> = [
+  Locales.enGB,
+  Locales.esES,
+  Locales.frFR,
+  Locales.itIT,
+]
+private let afterpayLocales: Set<Locale> = [
+  Locales.enAU,
+  Locales.enCA,
+  Locales.frCA,
+  Locales.enNZ,
+  Locales.enUS,
+  Locales.enUSposix,
```

Comment:
What's the difference between these two?

Reply:
The posix locale is generally used when formatting dates per [Apples developer docs](https://developer.apple.com/documentation/foundation/dateformatter) under "Fixed date representations". I don't believe this would be actively used as a configuration, but have updated to allow for this in [58c5787](https://github.com/afterpay/sdk-ios/pull/207/commits/58c5787b4c30506ac43d325c143a58c910bf5122)